test/integration/config-empty/test/index.test.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-env jest */
+/* global jasmine */
+import { join } from 'path'
+import { nextBuild } from 'next-test-utils'
+
+jasmine.DEFAULT_TIMEOUT_INTERVAL = 1000 * 60 * 5
+
+describe('Empty configuration', () => {
```

Comment:
This should also check development output 👍

Reply:
I added a check of stdout for successful compilation, is it what you meant and is it okay to keep them together? On my local machine each case running ±2-3sec, want to save some time 😅

Reply:
@ijjk could you add one?

Reply:
Added another check for development output

Reply:
Ah, thanks, now I see what it is :D